instant-segment/examples/contrived.rs

```diff
@@ -2,20 +2,20 @@ use instant_segment::{Search, Segmenter};
 use std::collections::HashMap;
 
 fn main() {
-    let mut unigrams = HashMap::default();
+    let mut unigrams = HashMap::new();
```

Comment:
This seems like an unrelated change? Prefer to avoid this in the same commit.

Reply:
Down below in that file we previously called `Segmenter::from_maps(unigrams, bigrams)` which fixes the hash builder type parameter of `unigrams` (and `bigrams`) to `ahash`. However, this is changed in this commit to call `Segmenter::new(unigrams, bigrams)` wich takes generic iterators as arguments and hence no type can be inferred for the hash builder . That's why we need to change from `HashMap::default` (which doesn't fix a hasher) to `HashMap::new`.

Reply:
Okay, fair enough!